test/alternator/test_encoding.py

```
@@ -309,3 +314,61 @@ def test_index_key_not_schema_column(dynamodb, cql, table_with_indexes):
+# for a table named {name}, with tablets explicitly enabled or disabled - so
+# that it differs from the keyspace Alternator would have created itself.
+# The explicit "initial" tablet count is needed because uses_tablets() below
+# cannot tell the automatic count (0) from vnodes.
+@contextmanager
+def precreated_keyspace(cql, this_dc, name, tablets):
+    tablets_opt = "{'enabled': true, 'initial': 1}" if tablets else "{'enabled': false}"
+    try:
+        cql.execute(f"CREATE KEYSPACE alternator_{name} WITH REPLICATION = "
+            f"{{'class': 'NetworkTopologyStrategy', '{this_dc}': 1}} "
+            f"AND TABLETS = {tablets_opt}")
+    except Exception as e:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)